SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1684,35 +1684,72 @@ USpatialActorChannel* ActorSystem::TryRestoreActorChannelForStablyNamedActor(AAc
 
 void ActorSystem::InvokeRepNotifies()
 {
-	for (FObjectRepNotifies& ObjectRepNotifies : RepNotifiesToSend)
+	// We want to call rep notifies on actors before subobjects
+	TArray<FObjectRepNotifies> Subobjects;
+	Subobjects.Reserve(RepNotifiesToSend.Num());
+
+	for (auto& ObjectToObjRepNotifies : RepNotifiesToSend)
```

Comment:
Object to Object rep notifies?
not sure about the name

Reply:
I was trying to communicate the key/value in the name, but maybe not needed - what would your preference for name be

Reply:
changed it now